### ISSUE TITLE
Jak pobierać prawidłowo dane z REST API allegro?

### ISSUE THREAD

**Issue by 13Dominik** [allegro/allegro-api#6820]
mam konto na allegro od dawna. Dzisiaj potwierdziłem logowanie dwu-etapowe i planowałem zacząć pracę z API. Po wielu próbach, udało się uzyskać access_token, (ma 799 znaków, nie wiem czy to poprawna wartość) jednak nie mogę dojść do porzadku z wysyłaniem prawidłowych requestów z nim. Nie chcę budować żadnych aplikacji stron zewnętrznych itp. jestem jedynie zainteresowany wysyłaniem requestów i wybieraniem z odpowiedzi JSON, opini o produkcie/produktach. Czy jest ktoś mi w stanie pomóc?

**Comment by Lukasz-Zurek** [comment]
Na wstępie zaznaczę, że w naszym API nie udostępniamy funkcjonalności dotyczącej pobierania opinii produktowych.

W pierwszej kolejności zapoznaj się proszę z [dokumentacją](https://developer.allegro.pl/documentation) Allegro API. Zachęcam również do skorzystania z naszego poradnika, np. [tutaj](https://developer.allegro.pl/tutorials/jak-jednym-requestem-wystawic-oferte-powiazana-z-produktem-D7Kj9gw4xFA#jak-znalezc-produkt) znajdziesz informacje dotyczące wyszukiwania produktów z naszego katalogu. 

Dla każdego zasobu opisujemy przykładowy request. W każdym z nich, w nagłówku **Authorization** przekazujesz access_token, czyli `Authorization: Bearer {token}`. Zwróć również uwagę na pozostałe nagłówki **Accept** i **Content-type** - w zależności od zasobu, z którego będziesz korzystał.

**Comment by Lukasz-Zurek** [comment]
Najlepiej będzie jak skorzystasz z [formularza kontaktowego](https://allegro.pl/pomoc/kontakt?kategoria=30db29ca-fb10-434b-a5f0-fc43019af49b). Być może uda się Tobie uzyskać dodatkowe informacje i dane na ten temat.

**Comment by 13Dominik** [comment]
Dzięki już napisałem w formularzu kontaktowym. Jeszcze mam jedno pytanie czy w przeszłości API Allegro udostępniało opinie?

**Comment by Lukasz-Zurek** [comment]
Nie, nie udostępnialiśmy wcześniej opinii produktowych w ramach Allegro API.
